### DISCUSSION TITLE
Request: Last day minus X

### DISCUSSION THREAD

**Discussion by giladno** [Hexagon/croner#221] [Ideas]
In many cron implementations, you can find the following syntax:

```
0 0 0 L-3 * *
```

Which basically means: 3 days before the last day of the month. Can this be added to `croner`?

**Comment by Hexagon** [comment]
Hello, thanks for the feedback 👋 

I'll convert this to an discussion, as we track requests over there.

**Comment by maliksenpai** [comment]


**Reply by Hexagon** [reply]
@maliksenpai Go for it 👍
